utils/utils.js

```diff
@@ -1,7 +1,25 @@
 const request = require('request');
 const errorPhrases = require('../assets/errorPhrases.json'); 
 
-asyncRequest = (url) => {
+asyncRequest = (url, spoofHeaders = false) => {
+
+    if (spoofHeaders) {
```

Comment:
A cleaner way of doing this might be something like 
```
const args = {}
if (spoofHeaders) {
  args['headers'] = ...
}

return new Promise((resolve, reject) => {
  request(url, args, (error, res, body)...)
}
```